app/components/all-books.js

```diff
@@ -1,6 +1,17 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
+
 
 export default Component.extend({
+
+  availableBooks: computed('book', function() {
```

Comment:
why is this a front end property? also (transactions.content) needs to be refactored to not use internal models. transactions are a promise so this needs to deal with the promise.